dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapAdapter.java

```diff
@@ -4119,4 +4182,32 @@ protected void setConfigurationAdmin(ServiceReference<ConfigurationAdmin> refere
     protected void unsetConfigurationAdmin(ServiceReference<ConfigurationAdmin> reference) {
         configAdminRef.unsetReference(reference);
     }
+
+    /**
+     * Get the configuration read lock.
+     */
+    private void getConfigReadLock() {
```

Comment:
Do you want to have the entry/exit trace on these methods? Or add `@Trivial` ?

Reply:
I do, simply to determine whether it plays a role in any performance issues when debugging.